src/main/java/uk/gov/hmcts/reform/pip/subscription/management/controllers/SubscriptionController.java

```diff
@@ -83,6 +86,12 @@ public ResponseEntity<String> createSubscription(
         @RequestHeader("x-user-id") String actioningUserId
     ) {
         Subscription subscription = subscriptionService.createSubscription(new Subscription(sub), actioningUserId);
+        // IF COURT LIST SUBSCRIPTION, ADD LIST AND LANGUAGE TYPE
+        if (subscription.getSearchType().equals(LOCATION_ID)) {
```

Comment:
I think this bit here is better done via a separate call via the FE to the configure lists endpoint below, rather than passed in as part of the subscriptions object.

Otherwise in the case of bulk adding subscriptions like selecting multiple courts, we'll need to send in the same config for each one, which would then re-run the configure lists for each of the subscriptions that get sent in.